Dockerfile.vcsim

```diff
@@ -1,4 +1,48 @@
+# Create a builder container
+FROM golang:latest as build
```

Comment:
Since we're talking security vulnerabilities here (+1 on this), can we please pin the builder image then via sha-digest as a best practice?

E.g.:

```Dockerfile
# golang:1.16.0-buster amd64
FROM golang@sha256:f254180c5defa2653955e963fb0626e3d4fbbb162f7cff6490e94607d1d867ff AS build
```

Reply:
complete.